ambry-api/src/main/java/com/github/ambry/quota/UserQuotaRequestCostPolicy.java

```diff
@@ -97,7 +114,7 @@ private double calculateCapacityUnitCost(RestRequest restRequest, RestResponseCh
           contentSize = restRequest.getBytesReceived();
         }
         double cost = Math.ceil(contentSize / CU_COST_UNIT);
-        return (cost > 0) ? cost : 1;
+        return Math.max(cost, MIN_CU_COST);
```

Comment:
now that we are here, we should probably fix the mistakes on stitch operation and others.
1. Stitch operation should cost "blob_size". it should cost min_cost_unit, which is 1.
2. If the POST operation is not for uploading blob, if it's for account or container, then it should only take min_cost_unit. 
3. We have to deal with PUT as well, since the Named blob uploads are using PUT.

Reply:
Will fix request cost logic in a follow up PR